packages/e2e/lib/e2e.dart

```diff
@@ -19,10 +19,16 @@ class E2EWidgetsFlutterBinding extends LiveTestWidgetsFlutterBinding {
     // TODO(jackson): Report test results as they arrive
     tearDownAll(() async {
       try {
-        await _channel.invokeMethod<void>(
-            'allTestsFinished', <String, dynamic>{'results': _results});
+        // For web integration tests we are not using the
```

Comment:
Maybe we should just check `!kIsWeb` and call `allTestsFinished` if it returns true? This plugin isn't federated yet so there isn't really a need to worry about future platforms, and it seems confusing to have a warning printed on every test run.

Reply:
done.